src/NUnitFramework/framework/Attributes/ValuesAttribute.cs

```diff
@@ -44,6 +45,7 @@ public class ValuesAttribute : NUnitAttribute, IParameterDataSource
         /// elements may have their type changed in GetData
         /// if necessary
         /// </summary>
+       
```

Comment:
I think this empty line should be removed, so that the comment is placed just on top of the the field.